Zajecia/src/main/java/pl/isa/javasmugglers/MainMenu.java

```diff
@@ -48,12 +49,20 @@ public void userSelection(int userSelection) {
                 newUser.addUser();
                 break;
             case 2:
-                //Tutaj wywołują swoje metody Mikołaj i Patryk
+
                 System.out.print("Wybrano 2");
+                UserLogin userLogin1 = new UserLogin();
+                String string1 = userLogin1.loginStudent().toString();
```

Comment:
Czym jest string1 userLogin1?

Reply:
W klasie User jest override metody to string, która wyrzuca tylko wartość pola userEmail, który jest elementem ścieżki pliku json z listą zajęć użytkownika.